### PR TITLE
Node consistency 91 100

### PR DESCRIPTION
**Pull request by martinstacey**
### Purpose
The following PR addresses nodes' names and descriptions to make them more descriptive for beginners.

**91) NODE: Image.WriteToFile**
-CHANGES:
Output Name: success
Output result= was not working, added a message: File has been succesfully written

**92) NODE: Image.Dimensions**
-CHANGES:
Output Descriptions: 
Width: width of image in pixels
Height: height of image in pixels

**93) NODE: FileSystem.WriteText**
-CHANGES:
Output Name: success
Output Description: Success message if file was written correctly 
Output result=  added a message: File has been succesfully written

**94) NODE: FileSystem.ReadText**
-CHANGES:
Output : str to string
InputDescription: file: File object to read text from

**95) NODE: Directory From Path**
-CHANGES:
Output Description : Path to the directory. (string)

**96) NODE: Directory.StringifyJSON**
-CHANGES:
Output Description : Path to the directory.

**97) NODE: Directory.ParseJSON**
-CHANGES:
Output Description: Path to the directory.

**98) NODE: Cuboid.Width**
-FYI Node located on protogeometry.dll Suggestion: Add description for cuboid input and revise overall description for clarity.

**99) NODE: Cuboid.Length**
-FYI Node located on protogeometry.dll Add description for cuboid input and revise overall description for clarity.

**100) NODE: Cuboid.Height**
-FYI Node located on protogeometry.dll Add description for cuboid input and revise overall description for clarity.

image

### Reviewers
@SHKnudsen
@StudioLE
@MarkThorley
@anrova

### FYIs
-Cuboid.Width, Cuboid.Length, Cuboid.Height: located on protogeometry.dll
-Icons added: 

image

### CODE REVIEW

**Comment by martinstacey** [comment]
Suggestions for nodes locked in protogeometry:
@Amoursol

**Cuboid.Width**
Input description: A cuboid
Output description: cuboid's width

**Cuboid.Length**
Input description: A cuboid
Output description: cuboid's length

**Cuboid.Height**
Input description: A cuboid
Output description: cuboid's height

PS: these may seem redundant descriptions but its less scary for beginners than the default `Autodesk.DesignScript.Geometry.Cuboid` description

**Review by Amoursol — APPROVED** [review]

File: src/Libraries/CoreNodeModels/Properties/Resources.Designer.cs [hunk 1]
```diff
@@ -286,7 +286,7 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Path to the directory..
+        ///   Looks up a localized string similar to type: string (Path to the directory).
```
Comment (aparajit-pratap) [inline src/Libraries/CoreNodeModels/Properties/Resources.Designer.cs:289]: Invert description and type. Type must follow description.

File: src/Libraries/CoreNodes/FileSystem.cs [hunk 2]
```diff
@@ -117,11 +118,13 @@ public static bool FileExists(string path)
...
         /// <param name="filePath">Path to write to</param>
         /// <param name="text">Text content</param>
+        /// <returns name="success"> Success message if file was written correctly </returns>
         /// <search>write file,text,file,filepath</search>
-        public static void WriteText(string filePath, string text)
+        public static string WriteText(string filePath, string text)
         {
             var fullpath = AbsolutePath(filePath);
             System.IO.File.WriteAllText(fullpath, text);
+            return "File has been succesfully written";
```
Comment (aparajit-pratap) [inline src/Libraries/CoreNodes/FileSystem.cs:127]: This is an API break. Please undo this change.

File: src/Libraries/CoreNodes/FileSystem.cs [hunk 3]
```diff
@@ -449,11 +454,12 @@ private static IEnumerable<byte> PixelsFromColor(Color color)
...
         /// <param name="path"></param>
         /// <param name="image">The image to write</param>
-        /// <returns name="ok">It is successful or not.</returns>
+        /// <returns name="success">Success message if file was written correctly </returns>
         /// <search>write image,image,file,filepath</search>
-        public static void WriteToFile(string path, Bitmap image)
+        public static string WriteToFile(string path, Bitmap image)
         {
             image.Save(FileSystem.AbsolutePath(path));
+            return "File has been succesfully written";
```
Comment (aparajit-pratap) [inline src/Libraries/CoreNodes/FileSystem.cs:462]: API break - please undo change.

**Review by aparajit-pratap — CHANGES_REQUESTED** [review]
Please address review comments.

File: src/Libraries/CoreNodeModels/Properties/Resources.Designer.cs (hunk 1 above)
Comment (martinstacey) [reply src/Libraries/CoreNodeModels/Properties/Resources.Designer.cs]: Adressed in 6ca33f7

File: src/Libraries/CoreNodes/FileSystem.cs (hunk 2 above)
Comment (martinstacey) [reply src/Libraries/CoreNodes/FileSystem.cs]: Addressed in a0e7b13

File: src/Libraries/CoreNodes/FileSystem.cs (hunk 3 above)
Comment (martinstacey) [reply src/Libraries/CoreNodes/FileSystem.cs]: Addressed in a0e7b13

**Review by aparajit-pratap — APPROVED** [review]
